packages/expo-notifications/src/NotificationsEmitter.ts

```diff
@@ -0,0 +1,20 @@
+import { EventEmitter, Subscription, NativeModulesProxy } from '@unimodules/core';
+
+type Notification = any;
```

Comment:
think it's worth creating TS type definitions for the serialized notifications in this module? 😬

Reply:
Done! Let me know if this makes sense. I also intend to parse those platform notifications into an easily-comprehensible notification like:

```ts
interface Notification {
  title: string | null;
  message: string | null;
  badge: string | null;
  body: { [key: string]: unknown };
  ios?: IosNotification;
  android?: IosNotification;
```

but for that I would need behavior specified in https://github.com/expo/expo/pull/6843 (since `title` in an `AndroidNotification` would be either `remoteMessage.notification.title` or `remoteMessage.data.title` as per our notification-building implementation.

Reply:
ok, that makes sense, let's revisit after #6843 lands then.